src/Essentials/src/Contacts/Contacts.uwp.cs

```diff
@@ -12,6 +12,10 @@ class ContactsImplementation : IContacts
 		public async Task<Contact> PickContactAsync()
 		{
 			var contactPicker = new ContactPicker();
+
+			var hwnd = WindowStateManager.Default.GetActiveWindowHandle(true);
+			WinRT.Interop.InitializeWithWindow.Initialize(contactPicker, hwnd);
```

Comment:
This is technically a "fix" for a crash, but the API still does not work with it and crashes again: https://github.com/microsoft/WindowsAppSDK/issues/1157